tips/tips-en.txt

```diff
@@ -89,3 +87,20 @@ https://www.haiku-os.org/docs/userguide/en/preferences/keymap.html
 GUI
 You can easily move windows between workspaces by dragging them around in the Workspaces applet. Switching through workspaces is even faster with the keycombo CTRL + ALT + cursor keys.
 https://www.haiku-os.org/docs/userguide/en/workspaces.html
+%
+Terminal
+Press CTRL R and start to enter a command and you'll be provided with the first match from the bash history. Keep pressing CTRL R until you find the right command line and press ENTER to execute it.
+https://www.haiku-os.org/docs/userguide/en/applications/terminal.html#shortcuts
+%
+GUI
+Use keyboard's PgUp/PgDown and Home/End to quickly navigate the context menu in a very crowded folder.
+https://www.haiku-os.org/docs/userguide/en/tracker.html
+%
+Miscellaneous
+You can open a document with an already running app by dragging its icon onto the apps' enty in the Deskbar.
+%
+Miscellaneous
+Hold down the right ALT+SHIFT+CRTL and click a program in the Deskbar to kill it.
```

Comment:
"ALT + SHIFT + CRTL". And for me, it only works on the left side of the keyboard...
Also add: "This is called 'Vulcan Death Grip':"